### DISCUSSION TITLE
How can I keep my klogg setting after update to version?

### DISCUSSION THREAD

**Discussion by kaje0105** [variar/klogg#439] [Q&A] [answered]
I am using klogg portable version. It is very useful for me. Thanks a lot.

I have a question, when I update my portable version klogg to new portable version,

how can I keep my settings? (font, highilight...)

Thanks!

**Comment by jeffjensen** [comment, marked as answer]
I also use portable as I find it the easiest way to use the latest continuous releases.  I simply copy klogg_session.conf and klogg.conf from the old installation dir to the new installation dir.

**Reply by kaje0105** [reply]
Thanks a lot!
